src/molecules/ContentBlock/ContentBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import cn from 'classnames';
+import { Banner, Heading, Button } from '../../index';
+
+export interface Block {
+  title: string;
+  text: string;
+  image: {
+    title: string;
+    url: string;
+  };
+  orientation: boolean;
+  buttonLink?: string;
+  buttonText?: string;
+}
+
+export interface ContentBlockProps extends Block {
+  className?: string;
+  search?: string;
+  whiteBackground?: boolean;
+}
+
+export const ContentBlock: React.FC<ContentBlockProps> = ({ className, block, search, whiteBackground = false }) => (
```

Comment:
If you want to keep the `block`, you can do:
```suggestion
export const ContentBlock: React.FC<ContentBlockProps> = ({ className, search, whiteBackground = false, ...block }) => (
```